content/en/docs/setup/getting-started/index.md

```diff
@@ -79,6 +79,12 @@ Follow these steps to get started with Istio:
     selected to have a good set of defaults for testing, but there are other
     profiles for production or performance testing.
 
+    {{< warning >}}
```

Comment:
An idea block may be preferable.

Reply:
IIUC, the demo profile doesn't work in OpenShift, so a warning is more appropriate.